src/common/camera_control.c

```diff
@@ -615,7 +615,7 @@ static void dt_camctl_camera_destroy(dt_camera_t *cam)
   gp_camera_unref(cam->gpcam);
   gp_widget_unref(cam->configuration);
 
-  for(GList *it = g_list_first(cam->open_gpfiles); it != NULL; it = g_list_delete_link(it, it))
+  for(GList *it = cam->open_gpfiles; it != NULL; it = g_list_delete_link(it, it))
```

Comment:
`it != NULL` can be just `it` for consistency with all other loops.

Some more instances below.